aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -85,24 +95,6 @@ impl AptosVMImpl {
         vm
     }
 
-    pub fn init_with_config(version: Version, gas_schedule: GasSchedule) -> Self {
```

Comment:
Note: I'm deleting this since no one is using it and it's been a hassle to maintain two different entry points